openapi.yaml

```diff
@@ -12021,6 +12021,23 @@ paths:
                 $ref: '#/components/schemas/Tag'
         default:
           $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+        - lang: Shell
+          source: >
+            curl -H "Content-Type: application/json" \
+                -H "Authorization: Bearer $TOKEN" \
+                -X POST -d '{
+                    "label": "example tag",
+                    "linodes": [123,456],
```

Comment:
@leslitagordita @yoshizzle I did some testing for this but ran into some problems:

When running the example command as it is here (with the IDs updated for resources that are actually on my account), the tag was created, but the Linodes and Volumes didn't get the tag applied to them. No error was returned by the API, and my API token had full permissions to write to my account. 

Can I ask you to try this out too? I'm not sure if this is a problem with the example I've added, or if it might be a bug in the API.

Reply:
I ran this with Linode and Volume IDs that exist on my account, and it worked as I expected.